src/PatientTestsApi/Test/Services/RetryCollection.spec.ts

```diff
@@ -22,7 +22,7 @@ describe("RetryCollection", async function (): Promise<void> {
 
     expect(result).is.equal(expectedResult);
     verify(mockCollection.insertOne(expectedDoc, anything())).thrice();
-    expect(elapsedTime).is.greaterThan(2000);
+    expect(elapsedTime).is.greaterThan(1999);
```

Comment:
Has this been failing? Guess with Javascript timing 1+1 does not necessarily equal 2. :)

Reply:
Yes this had been failing once in a lifetime, with the timing being exactly 2000.